test/spec/formmodel.spec.js

```diff
@@ -1201,9 +1206,9 @@ describe( 'merging an instance into the model', () => {
                 '<model xmlns:cc="http://cc.com"><instance><a><c>record</c><cc:meta xmlns:cc="http://cc.com"><cc:instanceID/></cc:meta></a></instance></model>'
             ],
             // namespaces used in both record and model (though now with triple equal namespace declarations..:
-            [ '<a xmlns:cc="http://cc.com"><c>record</c><cc:meta><cc:instanceID>a</cc:instanceID></cc:meta></a>',
-                '<model xmlns:cc="http://cc.com"><instance><a><c/><cc:meta><cc:instanceID/></cc:meta></a></instance></model>',
-                '<model xmlns:cc="http://cc.com"><instance><a xmlns:cc="http://cc.com"><c>record</c><cc:meta xmlns:cc="http://cc.com"><cc:instanceID>a</cc:instanceID></cc:meta></a></instance></model>'
+            [ '<a xmlns:orx="http://openrosa.org/xforms"><c>record</c><orx:meta><orx:instanceID>a</orx:instanceID></orx:meta></a>',
+                '<model xmlns:orx="http://openrosa.org/xforms"><instance><a><c/><orx:meta><orx:instanceID/></orx:meta></a></instance></model>',
+                '<model xmlns:orx="http://openrosa.org/xforms"><instance><a xmlns:orx="http://openrosa.org/xforms"><c>record</c><orx:meta xmlns:orx="http://openrosa.org/xforms"><orx:instanceID>a</orx:instanceID></orx:meta></a></instance></model>'
```

Comment:
GitHub won't let me comment inline on the [relevant lines](https://github.com/enketo/enketo-core/pull/836/files#diff-a3816804710c6de44954ba918725ac511d729dd2b01badd447b3d86e2c924951R1237-R1250), so noting here: in the future we should move these lines _into_ the `it` callback. If anything in those lines causes an error, it will terminate the `describe` block without adding the failing test, resulting in a false positive for the test run. (And we should do the same for any other logic executed outside `it`/setup/teardown callbacks. I just happened to notice this case in the course of review.)

Reply:
The link doesn't seem to work, but I think you're referring to e.g. initializing the model. That's a good point. Thanks! 👍